lib/debug.ml

```diff
@@ -0,0 +1,6 @@
+let debug fmt =
+  let pf = match Sys.getenv "CODEPT_DEBUG" with
+    | "1" | "true" -> Format.fprintf
+    | _ -> Format.ifprintf
+    | exception _ -> Format.ifprintf in
```

Comment:
`exception Not_found` ? I would rather not catch wandering asynchronous exception there.